### PR TITLE
DashboardView test added

### PR DESCRIPTION
**Pull request by quiz3**

### CODE REVIEW

**Comment by quiz3** [comment]
Once PR #104 is merged, please convert this PR from draft to a real PR and then review and merge it. It will work at that point in time.

File: src/main/java/app/testDashboardView.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        DashboardController dashboardController = new DashboardController(dashboardInteractor);
+
+        DashboardView dashboardView = new DashboardView(
+                dashboardViewModel,
+                dashboardController,
+                new BuyViewModel(),
+                new SellViewModel(),
+                new GetNewsViewModel(),
+                new ResetBalanceViewModel(),
+                new GetTransactionHistoryViewModel(),
```
Comment (awesominat) [inline src/main/java/app/testDashboardView.java:57]: Can you briefly explain why so many are needed as parameters?

Comment (awesominat) [reply src/main/java/app/testDashboardView.java:57]: ignore this comment, I asked this in the open PR #104

**Review by rickypramanick — APPROVED** [review]
Seems like the infinite API call issue has been addressed in this PR. Please be mindful of such erroneous calls in the future!

**Pull request merged**